src/class-wc-gateway-btcpay.php

```diff
@@ -973,14 +986,9 @@ public function ipn_callback()
                 $this->log('    [Info] Order ID is: ' . $order_id);
             }
 
-        //this is for the basic and advanced woocommerce order numbering plugins
-        //if we need to apply other filters, just add them in place of the this one
-            $order_id = apply_filters('woocommerce_order_id_from_number', $order_id);
```

Comment:
Will removing this break any installs? I know some have used this in the past, and that some have needed it.

Reply:
As far as I can see this is only relevant for the IPN communication. As new orders are created with the Order ID once this plugin is installed, the callback will also be received with that ID and there is no need anymore to convert it.

Reply:
Agree with Dennis here, as this is the ipn callback and we want to have the internal original order ID throughout all the payment process and BTCPay. What this does is that it will get the "display" order id. Which we do not want here anyway, this is only relevant for showing the order id on the invoice for the merchant and customer but for finding the correct order in the callback we need to store (in the invoice) and query with the original order id.

Reply:
Can confirm it works with and without custom order numbers.